### PR TITLE
EDGEORDERS-14 - Release  2.0.0 

### PR DESCRIPTION
**Pull request by KVupp**
## PURPOSE
https://issues.folio.org/browse/EDGORDERS-14 Create the 2.0.0 Release

## APPROACH
Update NEWS.md file
Upgrading the gobi dependency to 1.2, which the latest available version

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 2.0.0 - Released
```
Comment (craigmcnally) [inline NEWS.md]: Please see https://wiki.folio.org/display/FOLIJET/Acquisitions+-+Release+Procedures#Acquisitions-ReleaseProcedures-BestPractices

Stories should be in descending order, and a link to the diff/changlog should be provided (even if it won't be usable until the tag is actually created)

Comment (craigmcnally) [reply NEWS.md]: Also note the best practices section for naming of the release PR...

Comment (KVupp) [reply NEWS.md]: My Bad I missed the Best Practices section. Just looked at the Procedure. Done now

File: NEWS.md [hunk 2]
```diff
@@ -1,8 +1,17 @@
 ## 2.0.0 - Released
-* [EDGEORDERS-9](https://issues.folio.org/browse/EDGORDERS-9) - Upgrading to RAML 1.0
+
+Major changes for this release include confirming to the new API Key format changed in edge-common and creating a new /validate API to support both GET and POST Http
```
Comment (craigmcnally) [inline NEWS.md]: conforming

**Pull request merged**